test/strategy/ExtendConfigStrategyTest.js

```diff
@@ -7,6 +7,43 @@ var strategy = require('../../lib/strategy');
 var ExtendConfigStrategy = strategy.ExtendConfigStrategy;
 
 describe('ExtendConfigStrategy', function () {
+  it("should be able to extend member that references itself", function (done) {
+    var extendConfig = {
+      abc: "123"
```

Comment:
Haha, how would you search for a uuid in your code that is generated at runtime? I actually think it's clearer just to have it hard-coded in the tests. It's such a small thing and it's not really a value that you'd like to change.